torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,48 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<int64_t> l = pop(stack).toIntList();
+          for(const auto& elem: l) {
+            if(!elem){
+              push(stack, false);
+              return 0;
+            }
+          }
+          push(stack, true);
+          return 0;
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(float[] self) -> bool",
```

Comment:
nit: we use `self` to encode that an operator can be used as a method, so this would allow `li.all()` to work. doesn't super matter tho since no one would do that.